docs/pages/machine-id/troubleshooting.mdx

```diff
@@ -0,0 +1,286 @@
+ERROR: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none publickey], no supported methods remain
+
+ERROR: unable to execute tsh
+executing `tsh proxy`
+exit status 1
+
+kex_exchange_identification: Connection closed by remote host
+Connection closed by UNKNOWN port 65535
+```
+
+In particular, note the `ssh: unable to authenticate` message.
+
+### Explanation
+
+This can occur when attempting to log into the node as a user not listed as a
+principal on the SSH certificate.
+
+This can be verified by viewing the `tbot` logs and looking for the log message
+when impersonated certificates for the matching destination were renewed.
+
+In the following example, the only principal listed for the identity in
+`/opt/machine-id` is `alice` (via the `access` role):
+```text
+INFO [TBOT]      Successfully renewed impersonated certificates for directory /opt/machine-id, valid: after=2022-07-21T21:49:26Z, before=2022-07-21T22:50:26Z, duration=1h1m0s | kind=tls, renewable=false, disallow-reissue=true, roles=[access], principals=[alice -teleport-internal-join], generation=0 tbot/renew.go:630
+```
+
+... however, the SSH command attempted to login as `bob`.
+
+### Resolution
+
+Ensure the bot identity is allowed to login as the requested user by taking any
+of the following actions:
+
+ 1. Changing the SSH command to login as an allowed user
+ 2. Modifying the `access` role to allow the `alice` principal
+ 3. Adding an additional role granting login via the `bob` principal
+
+Note that if roles are added or modified, the certificates will need to be
+renewed for the changes to take effect. The bot will renew certificates on its
+own after the renewal interval (by default, 20 minutes), but you can trigger a
+renewal immediately by either restarting the `tbot` process or sending it a
+reload signal:
+
+```code
+$ pkill -sigusr1 tbot
+```
+
+## Database requests fail with `database "example" not found`, but the database exists
+
+### Symptoms
+
+When requesting [database access](../database-access/introduction.mdx)
+certificates, the certificate request fails with an error like the following:
+
+```text
+ERROR: Failed to generate impersonated certs for directory /opt/machine-id: database "example" not found
+database "example" not found
+```
+
+However, the database exists and can be seen by regular users via `tsh`:
+
+```code
+$ tsh db ls
+Name       Description Allowed Users Labels  Connect 
+---------- ----------- ------------- ------- ------- 
+example                [alice]       env=dev
+```
+
+### Explanation
+
+Unlike regular Teleport users, Machine ID bot users are granted only minimal
+Teleport [RBAC permissions](../access-controls/reference.mdx) and are not
+allowed to view or list databases by default unless granted permission via one
+or more roles. Users should grant these permissions to the bot via a role at
```

Comment:
I think we should remove the last sentence of this paragraph since the "Resolution" section provides this information with much more detail.